src/GameData.ts

```diff
@@ -55,6 +55,37 @@ interface SearchObjectContainer {
     score: number;
 }
 
+interface ChampionDatum {
+    id: number;
+    name: string;
+    key: string;
+    skins: string[];
+    type: string;
```

Comment:
You can make this field a string literal type so that TypeScript can infer exhaustiveness. 
```suggestion
    type: "ChampionDatum";
```